include/swift/Driver/ToolChain.h

```diff
@@ -305,6 +306,12 @@ class ToolChain {
   void addLinkRuntimeLib(const llvm::opt::ArgList &Args,
                          llvm::opt::ArgStringList &Arguments,
                          StringRef LibName) const;
+    
+  /// Validates arguments passed to the toolchain.
+  /// An override point for platform-specific subclasses to customize the
```

Comment:
Nitpick: Please put a blank line between the summary and the body of a doc comment. It shows up better in IDEs or Doxygen that way.